setup/send_assets_to_multi_giveaway_1.ts

```diff
@@ -1,141 +1,74 @@
-import fs from 'fs';
-import hre, {getNamedAccounts} from 'hardhat';
+/**
```

Comment:
Could this be generic ?
could the folder `multi_giveaway_1` be pass as an argument ?

Reply:
Done

Reply:
And i guess the file should be renamed ?
`send_assets_to_multi_giveaway_1.ts`